fuel/modules/fuel/assets/js/fuel/custom_fields.js

```diff
@@ -1297,8 +1297,6 @@ if (typeof(window.fuel.fields) == 'undefined'){
 					$(document).trigger('blockLoaded', [$this, context]);
 					$('#form').data('disabled', false);
 				});
-			} else {
-				$layout_fields.empty();
```

Comment:
Instead of deleting this line, I think you could move line 1287 to 1283 where it defines `$layout_fields` before the conditional logic.